edxval/tests/test_api.py

```diff
@@ -1386,3 +1386,30 @@ def test_video_image_urls_field_validation(self):
 
         self.assertEqual(len(exception_messages), 1)
         self.assertEqual(exception_messages.pop(), u'Must be a valid list of strings.')
+
+    def test_video_image_deletion_single(self):
+        """
+        Test video image deletion works as expected when an image is used by only the video being updated.
+        """
+        existing_image_name = self.course_video.video_image.image.name
+
+        api.update_video_image(self.edx_video_id, self.course_id, ImageFile(open(self.image_path2)), 'image.jpg')
+
+        with self.assertRaises(IOError) as file_open_exception:
```

Comment:
I would add a comment that this is need to test that image has been deleted from storage.